content/documentation/overview/index.adoc

```diff
@@ -67,13 +67,13 @@ image::app-metrics.png[Demo Website]
 
 === Workloads
 
-In the workload tab we have the list of workloads with their health, error rate and labels validations.
+In the Workloads tab we have the list of workloads with their health, error rate and labels validations.
```

Comment:
I suggest "Workloads tab displays the list of workloads...."

"...We have..." sounds less formal for docs.